geometric_features/aggregation/ocean/subbasin.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+
+import shapely
+import numpy
+import xarray
+import os
+import matplotlib.pyplot as plt
+import pyproj
+import gzip
+import shutil
+import tempfile
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.feature_collection import _round_coords
+
+from geometric_features.download import download_files
+from geometric_features.utils import write_feature_names_and_tags
+
+
+def subbasins(gf):
+    '''
+    Aggregates ocean regions into the following larger sub-basins:
+    * Arctic Ocean Basin
+    * North Atlantic Basin
+    * South Atlantic Basin
+    * North Pacific Basin
+    * South Pacific Basin
+    * Indian Ocean Basin
+    * Southern Ocean Basin
+    
+    Parameters
+    ----------
+        gf : ``GeometricFeatures``
```

Comment:
Oops, this is super minor but should be:
```suggestion
        gf : geometric_featues.GeometricFeatures
```
If we get it right, there will be an link in the documentation.